pyfar/dsp/dsp.py

```diff
@@ -1880,12 +1880,11 @@ def average(signal, mode='linear', axis=None, weights=None, keepdims=False):
             is reversed before returning the averaged signal.
 
         The default is ``'linear'``
-    axis: None, int, or tuple of ints, optional
-        Axis or axes along which the averaging is done. Can be ``None``, which
-        will calculate the average across all channel axes. Negative values
-        refer to ``signal.cshape`` to avoid averaging across the time or
-        frequency axis. If axis is a tuple of ints, average will perform on the
-        channels specified in the tuple. The default is ``None``.
+    caxis: None, int, or tuple of ints, optional
+        Channel axis or axes along which the averaging is done. The default
```

Comment:
```suggestion
        Channel axes along which the averaging is done. The default
```
Pl. alone should suffice here